n3fit/src/n3fit/checks.py

```diff
@@ -385,8 +385,8 @@ def check_consistent_parallel(parameters, parallel_models, same_trvl_per_replica
             "Replicas cannot be run in parallel with different training/validation "
             " masks, please set `same_trvl_per_replica` to True in the runcard"
         )
-    if parameters.get("layer_type") != "dense":
-        raise CheckError("Parallelization has only been tested with layer_type=='dense'")
+    if parameters.get("layer_type") == "dense_per_flavour":
+        raise CheckError("Parallelization has not been tested with layer_type=='dense_per_flavour'")
```

Comment:
Why this change? I think the previous one was more correct, though given there is another check that raises an error for anything but `dense` and `dense_per_flavour` it's effectively the same thing.